parsers/IN_WE.py

```diff
@@ -93,17 +98,46 @@ def fetch_data(
                 item[datetime_col] = dt.shift(minutes=1).floor("minute").datetime
             else:
                 item[datetime_col] = dt.floor("minute").datetime
-        df_data = pd.DataFrame(
-            [item for item in data if item[datetime_col].hour == dt_12_hour.hour]
-        )
-        return df_data
+        return data
```

Comment:
I'm thinking we can write the exception catching a little more concise:
```
try:
        data = json.loads(resp.json().get("d", {}))
except json.decoder.JSONDecodeError:
        raise ParserException(
            parser="IN_WE.py",
            message=f"{target_datetime}: {kind} data is not available",
        )
 datetime_col = KIND_MAPPING[kind]["datetime_column"]
 for item in data:
    ... do your stuff
 return data
    
```
This way no need for ```is_response_content_empty = False``` and the extra check at the end :)